dev-tools/integration/.env

```diff
@@ -1,4 +1,4 @@
-ELASTICSEARCH_VERSION=8.12.0-9d443b17-SNAPSHOT
+ELASTICSEARCH_VERSION=8.13.0-SNAPSHOT
```

Comment:
This change broke the automation with updatecli, see https://github.com/elastic/fleet-server/actions/runs/7197121542

is that the expected behaviour?

Reply:
sorry, I changed it to fix e2e tests, it was not working with a specific 8.13 build

Reply:
how can we fix this?

Reply:
Using  `8.13.0-jqmw70gd` in `main` and `8.12.0-8db20ea5` in `8.12`

I took those values from https://github.com/elastic/fleet-server/actions/runs/7197121542/job/19603532844#step:3:213

https://storage.googleapis.com/artifacts-api/snapshots/8.12.json
https://storage.googleapis.com/artifacts-api/snapshots/main.json

Reply:
do you mind raising prs to fix this or do you want me to?